.env

```diff
@@ -27,6 +27,17 @@ APP_SECRET=e248599aac0652bbae8be186ff450f6b
 # MAILER_DSN=smtp://localhost
 ###< symfony/mailer ###
 
+### default uri ###
+DEFAULT_URI=
+### default uri ###
+
+### OpenId config ###
+#OPEN_ID_PROVIDER_URL=
+#OPEN_ID_PROVIDER_CLIENT_ID=
+#OPEN_ID_PROVIDER_CLIENT_SECRET=
+OPEN_ID_PROVIDER_CACHE_PATH='%kernel.cache_dir%/.well_known_cache.php'
+### OpenId config ###
```

Comment:
These variables must be documented in the “Installation” section of the documentation (https://github.com/itk-dev/naevnssekretariatet/#installation). https://github.com/itk-dev/naevnssekretariatet/blob/develop/CHANGELOG.md should also be updated.